piwik-sdk/src/main/java/org/piwik/sdk/ecommerce/EcommerceItems.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Android SDK for Piwik
+ *
+ * @link https://github.com/piwik/piwik-android-sdk
+ * @license https://github.com/piwik/piwik-sdk-android/blob/master/LICENSE BSD-3 Clause
+ */
+
+package org.piwik.sdk.ecommerce;
+
+import android.support.annotation.Nullable;
+
+import org.json.JSONArray;
+import org.piwik.sdk.tools.CurrencyFormatter;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class EcommerceItems {
+    public Map<String, JSONArray> items = new HashMap<>();
+
+
```

Comment:
Not an issue, just FYI, with prefixing i meant "items" -> "mItems". Code formatting looks good now.